src/jit/codegenlegacy.cpp

```diff
@@ -22410,10 +22367,6 @@ void                CodeGen::genPInvokeCallEpilog(LclVarDsc *  frameListRoot,
     {
         if (retVal == RBM_INTRET || retVal == RBM_LNGRET)
         {
-#ifdef _TARGET_AMD64_
-            retTmp = compiler->tmpGetTemp(TYP_LONG);
-            inst_ST_RV(INS_mov, retTmp, 0, REG_INTRET, TYP_LONG);
-#elif defined(_TARGET_X86_)
```

Comment:
It's in the #else case of `_TARGET_ARM_`, which is the only other possibility in this file.

---
In reply to: [67235263](https://github.com/dotnet/coreclr/pull/5815#discussion_r67235263) [](ancestors = 67235263)